gitopia/chain.json

```diff
@@ -182,15 +182,44 @@
             "ics20-1"
           ]
         }
-      }
+      },
+        {
+        "name": "v4",
+        "recommended_version": "v4.0.0",
+        "compatible_versions": [
+          "v4.0.0"
+        ],
+        "proposal": 31,
+        "height": 24330422,
+        "cosmos_sdk_version": "v0.47.13",
+        "consensus": {
+          "type": "cometbft",
+          "version": "v0.37.6"
+        },
+        "ibc_go_version": "7.6.0",
+        "ics_enabled": [
+          "ics20-1"
+        ],
+        "next_version_name": "",
+        "sdk": {
+          "type": "cosmos",
+          "version": "v0.46.13"
```

Comment:
This version here (v0.46.13) doesn't match cosmos_sdk_version defined above (v0.47.13)